### DISCUSSION TITLE
Option to disable Flashlight dimming when doing sliders

### DISCUSSION THREAD

**Discussion by schiWasTaken** [ppy/osu#18634] [Ideas]
When doing sliders with the flashlight mod, the flashlight dims for as long as you're holding down the button. I would like an option (in Mod Customization) to disable this dimming effect because it's quite annoying (and it hurts my eyes) when you're playing maps with lots of fast short sliders. 

While Flashlight was never really designed for these fast maps, the added Flashlight Size setting made me want to try and use Flashlight on harder maps, and I would like to continue enjoying Flashlight.

**Comment by Rekunan** [comment]
I propose to instead remove this feature entirely, and then add an option to have a permanent customizable dim (as in change the dim %).

**Comment by Sr3qt** [comment]
I agree that the flashing lights are annoying. Adding on to what you said, there should also be a flashing lights warning when you use the flashlight mod.
